index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title> 
+    <link rel="stylesheet" href="style.css">
+
+    <!-- Link to FontAwesome page -->
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css" 
+    integrity="sha384-oS3vJWv+0UjzBfQzYUhtDYW+Pj2yciDJxpsK1OYPAYjqT085Qq/1cq5FLXAZQ7Ay"
+    crossorigin="anonymous"/>
+
+    <!-- Font link to font family: Playfair Display -->
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&display=swap" rel="stylesheet">
+    <!-- Font link to font family: Poppins -->
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    <!-- Hamburguer button -->
+    <nav class="nav-bar">
+        <button type="button" title="Menu" class="option-menu">
+            <i class="fas fa-bars"></i>
+        </button>
+    </nav>
+
+    <!-- Back ground photo 1 -->
+    <div class="contain-img1">
+        <img src="/assets/image_geometry_1.png" alt="Background photo" class="background-img1">
+    </div>
```

Comment:
- [OPTIONAL] To follow the best practice for HTML, I suggest using a semantic HTML tag `<header>` tag as a container of this section 🤝